README.md

```diff
@@ -1,83 +1,64 @@
-# msvc-code-analysis-action
+# Microsoft C++ Code Analysis Action
 
-This action enables code analysis to run while building a project with the Microsoft Visual C++ Compiler. The analysis
-will produce SARIF results that can be uploaded to the GitHub Code Scanning Alerts experience.
+This actions run code analysis for any CMake project built with the Microsoft Visual C++ Compiler. The analysis
+will produce SARIF results that can be uploaded to the GitHub Code Scanning Alerts experience and/or included as
+an artifact to view locally in the Sarif Viewer VSCode Extension.
 
 ## Usage
 
 ### Pre-requisites
 
 Include a workflow `.yml` file using an [example](#example) below as a template. Run the `msvc-code-analysis-action`
-before re-building your project using the appropriate operation mode detailed below.
+after configuring CMake for your project. Building the project is only required if the C++ source files involve the use
+of generated files.
```

Comment:
I had to read the last sentence twice before understanding it, you may want to rephrase it a little. "Building the project is only required if some of the C++ source files used in the build are also generated in the build."